src/status_im/chat/events/shortcuts.cljs

```diff
@@ -0,0 +1,38 @@
+(ns status-im.chat.events.shortcuts
+  (:require [status-im.ui.screens.wallet.send.events :as send.events]
+            [status-im.ui.screens.wallet.choose-recipient.events :as choose-recipient.events]
+            [status-im.ui.screens.navigation :as navigation]
+            [status-im.utils.ethereum.core :as ethereum]))
+
+;; TODO(goranjovic) - update to include tokens in https://github.com/status-im/status-react/issues/3233
+(defn- transaction-details [contact]
+  (-> contact
+      (select-keys [:name :address :whisper-identity])
+      (assoc :symbol :ETH
+             :gas (ethereum/estimate-gas :ETH)
+             :from-chat? true)))
+
+(defn send-shortcut-fx [db contact params]
+  {:db         (-> db
+                   (send.events/set-and-validate-amount-db (:amount params))
+                   (choose-recipient.events/fill-request-details (transaction-details contact))
+                   (navigation/navigate-to :wallet-send-transaction-chat))
+   :dispatch-n [[:wallet/update-gas-price]]})
```

Comment:
Why this dispatch, is `:wallet/update-gas-price` not exposed as regular `:fx` producing function ? If not, it probably should be.